pytest/tests/mocknet/get-profile.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```

Comment:
Consider putting it to `mocknet/helpers` folder, my idea is that can be a standard set of tools which every mocknet node must have.